### PR TITLE
[11주차]_조민주

### PR DESCRIPTION
**Pull request by mummy-alive**
[필기](https://lateral-pastry-47c.notion.site/11-Hyperparam-Tuning-Batch-normalization-c60bbec47c394f658536097dd9dc3c4a?pvs=4)

[캐글필사](https://colab.research.google.com/drive/1Fq07aVRirq4HEUsISfLtw8nDqEFKBmxY?usp=sharing) 이번주도 감사합니다!

### CODE REVIEW

**Comment by bsoiyou** [comment]
11주차도 고생하셨습니다~!😊
